autoextract/aio/retry.py

```diff
@@ -95,13 +105,22 @@ def __call__(self, retry_state: RetryCallState) -> float:
             return self.network_wait(retry_state=retry_state)
         elif _is_server_error(exc):
             return self.server_wait(retry_state=retry_state)
+        elif _is_retriable_query_error(exc):
+            try:
+                wait = exc.retry_seconds
+            except ValueError as exc:
+                logger.warning(exc)
```

Comment:
```python
    raise ValueError(
                f"Could not extract retry seconds "
                f"from Domain Occupied error message: {self.message}"
            )
```

Reply:
Instead of raising an exception, we're logging a warning message and returning a default value (5 minutes).
The code that previously relied on `QueryError.retry_seconds` will no longer need to capture `ValueError` exceptions.